src/test/scala/uk/gov/nationalarchives/LambdaTest.scala

```diff
@@ -39,167 +33,14 @@ class LambdaTest extends AnyFlatSpec with MockitoSugar with BeforeAndAfterEach {
-      )
-    }
-  }
-
-  private def checkDynamoItems(tableRequestItems: List[DynamoTableItem], expectedTable: DynamoTable) = {
-    val items = tableRequestItems
-      .filter(_.PutRequest.Item.items("id").asInstanceOf[DynamoSRequestField].S == expectedTable.id.toString)
-      .map(_.PutRequest.Item)
-    items.size should equal(1)
-    val dynamoFieldItems = items.head.items
-    def list(name: String): List[String] = dynamoFieldItems
-      .get(name)
-      .map(_.asInstanceOf[DynamoLRequestField].L)
-      .getOrElse(Nil)
-    def strOpt(name: String) = dynamoFieldItems.get(name).map(_.asInstanceOf[DynamoSRequestField].S)
-    def str(name: String) = strOpt(name).getOrElse("")
-    str("id") should equal(expectedTable.id.toString)
-    str("name") should equal(expectedTable.name)
-    str("title") should equal(expectedTable.title)
-    expectedTable.id_Code.map(id_Code => str("id_Code") should equal(id_Code))
-    str("parentPath") should equal(expectedTable.parentPath)
-    str("batchId") should equal(expectedTable.batchId)
-    str("description") should equal(expectedTable.description)
-    dynamoFieldItems.get("fileSize").map(_.asInstanceOf[DynamoNRequestField].N) should equal(expectedTable.fileSize)
-    str("type") should equal(expectedTable.`type`.toString)
-    strOpt("customMetadataAttribute1") should equal(expectedTable.customMetadataAttribute1)
-    strOpt("customMetadataAttribute2") should equal(expectedTable.customMetadataAttribute2)
-    strOpt("attributeUniqueToBagInfo") should equal(expectedTable.attributeUniqueToBagInfo)
-    list("originalFiles") should equal(expectedTable.originalFiles)
-    list("originalMetadataFiles") should equal(expectedTable.originalMetadataFiles)
-  }
-
-  case class IngestMapperTest() extends Lambda {
-    val creds: StaticCredentialsProvider = StaticCredentialsProvider.create(AwsBasicCredentials.create("test", "test"))
-    private val asyncS3Client: S3AsyncClient = S3AsyncClient
-      .crtBuilder()
-      .endpointOverride(URI.create("http://localhost:9003"))
-      .credentialsProvider(creds)
-      .region(Region.EU_WEST_2)
-      .build()
-    private val asyncDynamoClient: DynamoDbAsyncClient = DynamoDbAsyncClient
-      .builder()
-      .endpointOverride(URI.create("http://localhost:9005"))
-      .region(Region.EU_WEST_2)
-      .credentialsProvider(creds)
-      .build()
-    val uuidsIterator: Iterator[String] = uuids.iterator
-    override val metadataService: MetadataService = new MetadataService(DAS3Client[IO](asyncS3Client))
-    override val dynamo: DADynamoDBClient[IO] = new DADynamoDBClient[IO](asyncDynamoClient)
-    override val randomUuidGenerator: () => UUID = () => UUID.fromString(uuidsIterator.next())
-  }
   "handleRequest" should "return the correct values from the lambda" in {
-    val (folderIdentifier, assetIdentifier, _, _, _, _) = stubValidNetworkRequests()
+    val testUtils = new LambdaTestTestUtils(dynamoServer, s3Server, discoveryServer)
```

Comment:
Similar comment to the one on https://github.com/nationalarchives/dr2-ingest-asset-opex-creator/pull/54
You can just create this once and then import it to cut down on the duplication.